src/parser/parse_tree_cl.c

```diff
@@ -5030,6 +5039,8 @@ pt_init_apply_f (void)
   pt_apply_func_array[PT_COMMIT_WORK] = pt_apply_commit_work;
   pt_apply_func_array[PT_CREATE_ENTITY] = pt_apply_create_entity;
   pt_apply_func_array[PT_CREATE_INDEX] = pt_apply_create_index;
+  pt_apply_func_array[PT_CREATE_HISTOGRAM] = pt_apply_create_histogram;	//TODO
+  pt_apply_func_array[PT_DROP_HISTOGRAM] = pt_apply_create_histogram;
```

Comment:
```suggestion
  pt_apply_func_array[PT_DROP_HISTOGRAM] = pt_apply_drop_histogram;
```
`pt_apply_drop_histogram`도 추가가 필요한거죠?

Reply:
아니요 불필요합니다. 재활용 가능한 함수입니다. 주석 제거하겠습니다.